arrays/LC128_longestConsecutive/index.js

```diff
@@ -1,3 +1,6 @@
+// This is an exclusive problem only available in the Complete and Deluxe versions of this course.
```

Comment:
This is the type of comment that got under my skin -- I'd strongly suggest you update your Udemy site to reflect this.